### PR TITLE
Cut #6.500, #6.502 in FromCOSE

### PR DESCRIPTION
**Pull request by deeglaze**
Top level tags are not interpretable as a COSE object. The protected header content-type is not meant to be application/rim+cbor according to the spec draft (Issue #132).

Support the spec's optional unsigned corim tag in COSE payload.

### CODE REVIEW

**Review by thomas-fossati — APPROVED** [review]
LGTM, thanks!

File: corim/signedcorim.go [hunk 1]
```diff
@@ -15,7 +16,8 @@ import (
 )
 
 var (
-	ContentType          = "application/rim+cbor"
+	ContentType          = "application/unsigned-corim+cbor"
```
Comment (thomas-fossati) [inline corim/signedcorim.go:19]: The media types defined by CoRIM are:

* [application/corim-signed+cbor](https://ietf-rats-wg.github.io/draft-ietf-rats-corim/draft-ietf-rats-corim.html#sec-mt-corim-signed)
* [`application/corim-unsigned+cbor`](https://ietf-rats-wg.github.io/draft-ietf-rats-corim/draft-ietf-rats-corim.html#sec-mt-corim-unsigned)

Comment (deeglaze) [reply corim/signedcorim.go]: Fixed.

**Comment by deeglaze** [comment]
I don't have permissions to merge. Did we decide I'd become a maintainer in the Veraison call? Or did I just suggest it? @yogeshbdeshpande

**Comment by thomas-fossati** [comment]
> I don't have permissions to merge. Did we decide I'd become a maintainer in the Veraison call? Or did I just suggest it? @yogeshbdeshpande

I made you an admin to both `veraison/corim` and `veraison/cocli`.

**Comment by yogeshbdeshpande** [comment]
> > I don't have permissions to merge. Did we decide I'd become a maintainer in the Veraison call? Or did I just suggest it? @yogeshbdeshpande
> 
> I made you an admin to both `veraison/corim` and `veraison/cocli`.

@deeglaze : to your question: _Did we decide I'd become a maintainer in the Veraison call?_

Yes certainly, as I said in the call, your contributions to Veraison project are highly Welcome & Appreciated!

**Comment by yogeshbdeshpande** [comment]
@deeglaze: Thank you for your changes, look good to me.!

This change also demands a suggested change in the `veraison/docs` repository, as we would recommend the usage of `Content-type` to be in-line with the CoRIM draft. If you are ok, then please also change:
https://github.com/veraison/docs/blob/main/api/endorsement-provisioning/README.md

and also the associated `yaml` file in the same location. Note: I have added you to that repo so you should be able to submit!

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Comment by thomas-fossati** [comment]
> @deeglaze: Thank you for your changes, look good to me.!
> 
> This change also demands a suggested change in the `veraison/docs` repository, as we would recommend the usage of `Content-type` to be in-line with the CoRIM draft. If you are ok, then please also change: https://github.com/veraison/docs/blob/main/api/endorsement-provisioning/README.md

This is not strictly connected.  The `Content-Type` in the docs is already out of sync with Veraison's implementation and should've been fixed irrespective of this change.

> and also the associated `yaml` file in the same location. Note: I have added you to that repo so you should be able to submit!

If Dionna wants to do it, great, but she should not feel obligated to do it :-)

**Comment by yogeshbdeshpande** [comment]
> > @deeglaze: Thank you for your changes, look good to me.!
> > This change also demands a suggested change in the `veraison/docs` repository, as we would recommend the usage of `Content-type` to be in-line with the CoRIM draft. If you are ok, then please also change: https://github.com/veraison/docs/blob/main/api/endorsement-provisioning/README.md
> 
> This is not strictly connected. The `Content-Type` in the docs is already out of sync with Veraison's implementation and should've been fixed irrespective of this change.
> 
> > and also the associated `yaml` file in the same location. Note: I have added you to that repo so you should be able to submit!
> 
> If Dionna wants to do it, great, but she should not feel obligated to do it :-)



> > @deeglaze: Thank you for your changes, look good to me.!
> > This change also demands a suggested change in the `veraison/docs` repository, as we would recommend the usage of `Content-type` to be in-line with the CoRIM draft. If you are ok, then please also change: https://github.com/veraison/docs/blob/main/api/endorsement-provisioning/README.md
> 
> This is not strictly connected. The `Content-Type` in the docs is already out of sync with Veraison's implementation and should've been fixed irrespective of this change.
@thomas-fossati I am referring to content-type as `Content-Type: application/rim+cbor` which we is part of documentation and also in CoRIM repo and also in apiclient repo
> 
> > and also the associated `yaml` file in the same location. Note: I have added you to that repo so you should be able to submit!
> 
> If Dionna wants to do it, great, but she should not feel obligated to do it :-)

**Comment by deeglaze** [comment]
I've removed the change to the content type, so this should be purely ergonomics right now.

**Pull request merged**